scripts/publish.rs

```diff
@@ -52,6 +52,7 @@ const CRATES_TO_PUBLISH: &[&str] = &[
     "wasmtime-fiber",
     "wasmtime-environ",
     "wasmtime-runtime",
+    "wasmtime-cranelift-shared",
```

Comment:
@alexcrichton @saulecabrera let me know if this works to add this here to pass merge queue CI, or if we should be adding `publish = false` to the new `Cargo.toml` file

Reply:
Yeah this is the expected fix